### ISSUE TITLE
Missing sideEffects option for @fortawesome/pro-thin-svg-icons package (v6)

### ISSUE THREAD

**Issue by Schinkentanz** [FortAwesome/Font-Awesome#18392]
Hello there, the current version (`6.0.0-beta2`) of the `@fortawesome/pro-thin-svg-icons` does not have the 
`"sideEffects": false` option set in the `package.json` file, wich leads to unnecessary large build sizes due to missing tree shaking. To workaround this issue, an icon can be included with a deep import:
```js
import { faFile } from '@fortawesome/pro-thin-svg-icons/faFile';
```
For all other packages (light, duotone, regular & solid) the `sideEffects` option is set correctly 👍 
![image](https://user-images.githubusercontent.com/1647548/138864334-d50a4440-4a32-42f9-9688-1111a337147b.png)
Let me know if I can provide more information on this.

**Comment by robmadole** [comment]
@Schinkentanz this should be fixed in 6.0.0-beta3. Can you give it a try?

**Comment by Schinkentanz** [comment]
@robmadole Works like a charm! Thanks for the fix
